core/state/database.go

```diff
@@ -227,6 +230,7 @@ func (db *cachingDB) StartVerkleTransition(originalRoot, translatedRoot common.H
                                                     |__|`)
 	db.CurrentTransitionState = &TransitionState{
 		started: true,
+		ended:   false,
```

Comment:
We need now to be more strict with initialization here, sine remember there's the existing call `bc.stateCache.InitTransitionStatus(true, true)` mentioned in the first comment of this PR.